avm/utilities/pipelines/staticValidation/psrule/ps-rule.yaml

```diff
@@ -81,3 +81,4 @@ rule:
     - Azure.VM.UseHybridUseBenefit
     - Azure.Storage.UseReplication
     - Azure.AppConfig.PurgeProtect
+    - Azure.APIM.MultiRegion
```

Comment:
What was ther reson for this? We should probably start attaching comments with the reason to every suppression

Reply:
we agreed in Teams to disable this one. It's untenable for most deployments to use multiregion deployment for APIM in every case ($$$), despite it being recommended.

Reply:
Hey Tony, the reason is absolutely fine. If possible please add it as a comment at the end of the line. Something we should really start doing for later reference 🙂